serverless.yml

```diff
@@ -63,3 +80,53 @@ package:
   exclude:
     - .webpack
 
+
+# Reference: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-product-property-reference.html
+resources:
+   Resources:
+      WebsiteDistribution:
+        Type: 'AWS::CloudFront::Distribution'
+        Properties:
+          DistributionConfig:
+            DefaultCacheBehavior:
+              TargetOriginId: 'ApiOrigin'
+              ViewerProtocolPolicy: 'redirect-to-https'
+              MinTTL: 0
+              AllowedMethods:
+                - GET
+                - HEAD
+                - OPTIONS
+                - POST
+                - PUT
+                - PATCH
+                - DELETE
```

Comment:
I wonder if we can do `- *` here...

Reply:
Doesn't seem possible according to the docs https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cloudfront-distribution-cachebehavior.html#cfn-cloudfront-distribution-cachebehavior-allowedmethods